user-manager/src/test/java/com/qpay/usermanager/controller/CustomerControllerIntegrationTest.java

```diff
@@ -0,0 +1,204 @@
+package com.qpay.usermanager.controller;
+
+import com.fasterxml.jackson.databind.ObjectMapper;
+import com.qpay.usermanager.model.dto.customer.CustomerModification;
+import com.qpay.usermanager.model.entity.customer.CustomerEntity;
+import com.qpay.usermanager.service.exception.CustomerNotFoundException;
+import com.qpay.usermanager.service.impl.CustomerServiceImpl;
+import com.qpay.usermanager.utility.Paths;
+import org.junit.jupiter.api.Test;
+import org.springframework.beans.factory.annotation.Autowired;
+import org.springframework.boot.test.autoconfigure.web.servlet.WebMvcTest;
+import org.springframework.boot.test.mock.mockito.MockBean;
+import org.springframework.http.HttpStatus;
+import org.springframework.http.MediaType;
+import org.springframework.test.web.servlet.MockMvc;
+import org.springframework.test.web.servlet.request.MockMvcRequestBuilders;
+
+import static org.assertj.core.api.Assertions.assertThat;
+import static org.mockito.BDDMockito.given;
+
+
+@WebMvcTest(controllers = CustomerController.class)
+class CustomerControllerIntegrationTest {
+
+    @Autowired
+    private MockMvc mockMvc;
+
+    @Autowired
+    private ObjectMapper objectMapper;
+
+    @MockBean
+    private CustomerServiceImpl customerService;
+
+    private final CustomerEntity customerEntity = CustomerEntity
```

Comment:
Constants should be written in uppercase

Reply:
https://github.com/Hoocki/qpay/pull/11#discussion_r1432524694

Reply:
Kostya is right. It is a constant.  Strings, numbers, booleans, objects (dto, entities) are constants.
Services, mappers, controllers and other components are not